client/state/themes/selectors/get-recommended-themes-filter.js

```diff
@@ -1,16 +1,13 @@
-import isSiteUsingCoreSiteEditor from 'calypso/state/selectors/is-site-using-core-site-editor';
-import 'calypso/state/themes/init';
+import { isCoreFSEEligible } from 'calypso/state/gutenberg-fse-settings/selectors';
 
 /**
  * We currently support two types of theme filtering for recommended themes,
  * with the possibility of more in the future, which will require some
  * further work in this selector.
  *
- * @param {object}      state  Global state tree
  * @param {number|null} siteId Site ID
  * @returns {string}           The string to filter on in an API request.
  */
-export function getRecommendedThemesFilter( state, siteId ) {
-	const isUsingSiteEditor = isSiteUsingCoreSiteEditor( state, siteId );
-	return isUsingSiteEditor ? 'block-templates' : 'auto-loading-homepage';
+export function getRecommendedThemesFilter( siteId ) {
```

Comment:
I know we have a convention of adding `state` as the first argument, but this is not the case here for two reasons:

1. I'm not using `state` at all;
2. It doesn't make sense since the cached state is actually coming from `react-query` and not `redux`.